.github/workflows/CI.yml

```diff
@@ -59,14 +59,51 @@ jobs:
         run: yarn circuits circuit:setup:plonk
 
       - name: Relayer Tests
-        run: yarn relayer test
+        run: |
+          yarn relayer test
+          echo "TESTNET_RELAYER_ACCOUNT_ID=$TESTNET_RELAYER_ACCOUNT_ID" >> $GITHUB_OUTPUT
+          echo "TESTNET_RELAYER_PRIVATE_KEY=$TESTNET_RELAYER_PRIVATE_KEY" >> $GITHUB_OUTPUT
+          echo "TESTNET_HYC_CONTRACT=$TESTNET_HYC_CONTRACT" >> $GITHUB_OUTPUT
 
       - name: Rust Tests
         run: yarn contracts test:rust
 
       - name: Lib Tests
         run: yarn contracts test:lib
 
+      - name: Building Relayer
+        run: yarn relayer build
+
+      - name: Log Envs
```

Comment:
Couldn't this be removed after debugging?

Reply:
I add this logs for us check everything's if we need